conda-forge.yml

```diff
@@ -1,6 +1,3 @@
-#bot:
-#  abi_migration_branches:
-#  - 2.26.x
```

Comment:
I had purposefully left this in here as a reminder. It survives rerendering. Frustratingly it was the bot commit itself that removed it. I'm surprised since AFAIU there is no prohibition to adding comments to `conda-forge.yml`.